external/batch-import.php

```diff
@@ -0,0 +1,63 @@
+<?php
+ini_set('display_errors', 1);
+if (!defined('XHGUI_ROOT_DIR')) {
+    require dirname(__DIR__) . '/src/bootstrap.php';
+}
+
+$sites      = new Xhgui_Sites();
```

Comment:
I find avoiding space alignment entirely saves time and annoyance in the long run :smile: